index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <main>
+        <h1>Pavel Ptashynski (Junior Frontend Developer)</h1>
+        <img src="1.jpg" alt="photo">
```

Comment:
In the future give meaningful names to images (to whatever assets: icons, images and so on) and alt tags. Check examples on some popular and trustable sites.